xmodule/tests/test_capa_block.py

```diff
@@ -725,6 +725,110 @@ def test_submit_problem_correct(self):
         # and that this was considered attempt number 2 for grading purposes
         assert block.lcp.context['attempt'] == 2
 
+    @patch('xmodule.capa.correctmap.CorrectMap.is_correct')
+    @patch('xmodule.capa_block.ProblemBlock.get_problem_html')
+    def test_submit_problem_correct_last_score(self, mock_html: Mock, mock_is_correct: Mock):
+        block = CapaFactory.create(attempts=0, max_attempts=2, grading_method='last_score')
+        mock_html.return_value = "Test HTML"
```

Comment:
Can we test that not configuring the last_score means that would be used as the default?

Reply:
In this test instead of passing the grading method explicitly, we can ignore it, so we verify that the default value is `last_score`